Blog post for my experience with ALiAS (#45)
asetalias/blog: content/post/My-journey-with-ALiAS.md

+++
title = "My journey with ALiAS : From a commit to the community"

> slug = "FOSS communities"

Reviewer: Slugs don't work well with capitals and spaces. How about something unique like `animeshaliasjourney`

Author: > Slugs don't work well with capitals and spaces. How about something unique like `animeshaliasjourney`

I guess `alias journey` would be simpler and better.

Reviewer: A slug should be self-explanatory if a person just reads it from the URL. 
Since there will be a lot of journey posts hence it's better to name them 

Better Slud - animesh-alias-journey

Author: > Better Slug - `animesh-alias-journey`

LGTM

image = "/images/community.jpg"
date = "2019-05-25T00:00:00"

> tags='ALiAS, FOSS'

Reviewer: ```suggestion
tags=['ALiAS', 'FOSS', 'Experiences', 'memories', `Alumni`]
```

Author: > Good stuff, thanks for writing this blog post. Some changes that are needed and review made inline
> 
> * Run the content through Grammarly for tweaking the tone.

Yeah, apparently I have this bad habit of intentionally omitting articles while writing/typing to save time :P. Gotta fix this.

> * You can add images of your time with ALiAS, meetups, our events, hackathons, prizes stuff that inspires people further.

Yeah, that would take some time to extract pics from social profiles of ALiAS, since I don't have any personal collection of those.

> * Why not write more? This is a great start. What say?
Yeah that's the plan. I'm just getting started.

Reviewer: Well, picking one to pictures won't be that hard. 
How about adding some of ILuGD or LinuxChix meetups that you went to.

Author: > Well, picking one to pictures won't be that hard.
> How about adding some of ILuGD or LinuxChix meetups that you went to.

I guess PyDelhi or mega meetups pics would better for this, since they had more attendance from ALiAS so those pics would resonate more with my ALiAS journey. What say?

Reviewer: Go for it, I saw a picture of a recent Ilugd meetup with you in it. Put that as well.

Reviewer: ?

Author: Done.

> tags=['ALiAS, FOSS, Alumni, Experiences']

Reviewer: ```suggestion
tags=['ALiAS', 'FOSS', 'Alumni', 'Experiences']
```

Reviewer: ```suggestion
tags=['ALiAS', 'FOSS', 'Alumni', 'Experiences']
```

> categories = 'Shared experience'

Reviewer: ```suggestion
categories = 'Alumni journals'
```
Since, we want to keep all Alumni articles in one category.

Reviewer 2: more like
```suggestion
categories = ['Shared experience']
```

Author: Good one.

> categories = ['Alumni Journals']

Reviewer: ```suggestion
categories = ['Alumni-Journals']
```
The blog is facing some issues with 2 word strings, hence a workaround. 
CC: @Sc0rpi0n101

Author: Done.

> description = "The encapsulation of my time with ALiAS club"

Reviewer: ```suggestion
description = "The encapsulation of my time with the ALiAS community/family"
```
One of the two.

Author: Done.

So going from a small club revived after years, to a huge community of people like me, the end of which can't (and shouldn't) be defined, ALiAS helped me to bring a new spark in my life. And I can say that no one or nothing can put out this spark, ever.

## Part of the journey is the end, but not for us ....... not us
Speaking in the present time, my college life had reached its end. All those moments in four years are now just memories. As the universal truth is, that everything comes to an end somewhere at some time. But if you ask the same for my relationship with ALiAS, I'll say, "Not yet, we're still on."

ALiAS has grown beyond a club or a community. It's now more like a part of us. It's inseparable from us. As long as we go on, it goes on with us. This might not be true for everyone else, but it is for me. And I say this as an active member and contributor of Amity Linux Assistance Sapience a.k.a. ALiAS :P.

> 

Reviewer: A conclusion is missing, what we like to know is definitive things you got from ALiAS, where are the pictures? 
This could be the Paytm hack you participated in, talks, meetups with ILuGD. This is good article, help us make this great.

And if that wasn't enough, getting along with the fellow ALiASians to various FOSS meetups, devsprints, hackathons was another level of experience. Victory for us wasn't about winning in the events, but leaving our impression on people around us. And I can say, we did win in that case.

So going from a small club revived after years, to a huge community of people like me, the end of which can't (and shouldn't) be defined, ALiAS helped me to bring a new spark in my life. And I can say that no one or nothing can put out this spark, ever.

## FOSS Meetups, projects, hackathons, and a lot more...
ALiAS gave me a big motivation to get into various FOSS meetups, hackathons, etc. and get involved in their community projects too. Along with learning and experience, all of those connected me to a wider band of people from different regions and professions, and like always the fun moments made to the top. Some honourable mentions among those are:

- Web projects of ALiAS, including its official website.
- Community meetups of PyDelhi, ILUG-D, LinuxChix India etc.

{{< figure src="/blog/images/megameet.jpg" caption="With my ALiAS mates at FOSS mega meetup at Adobe, Noida" >}}

> {{< figure src="/blog/images/ILUGDmeet.jpg" caption="Recent ILUG-D meetup at The Office Pass, Gurugram" >}}

Reviewer: Please follow the markdown formatting rules for adding images. Also, we would really appreciate if images could be stored off in a remote resource and URL's are added as the image source. The repo size is increasing constantly and we like to keep it minimum.

Author: Done. Check now
